Sources/Algorithms/Unique.swift

```diff
@@ -30,8 +39,22 @@ extension Sequence where Element: Hashable {
 
 extension Sequence {
   /// Returns an array with the unique elements of this sequence (as determined
-  /// by the given projection), in the order of the first occurence of each
+  /// by the given projection), in the order of the first occurrence of each
   /// unique element.
+  ///
+  ///     let animals = ["dog", "pig", "cat", "ox", "dog","cat"]
+  ///     let uniqued = animals(on: { $0.first })
+  ///     print(uniqued)
+  ///     // Prints "["dog", "pig","cat", "ox"]"
```

Comment:
```suggestion
  ///     // Prints '["dog", "pig", "cat", "ox"]'
```

Although the output depends on what the projection chosen for this example is.